src/hooks/useSound.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useRef } from 'react';
+
+export const useSound = (src: string) => {
```

Comment:
[imo]
interfaceは切って疎結合にしておくのと、ついでにローディングやエラーの状態は返して、使用する側でハンドリング（ローディング中ならぐるぐるさせるとか）はしたほうが良さそうです。
※ローカルだと特に問題ないですが、実際のユーザーのネットワーク環境が良い保証はないため

以下はイメージですmm
```suggestion
interface UseSoundResult {
  play: () => void;
  isLoading: boolean;
  error: Error | null;
}

export const useSound = (src: string): UseSoundResult => {
```